packages/terra-date-picker/src/terra-dev-site/test/date-picker/DatePickerFieldDefault.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import DatePickerField from '../../../DatePickerField';
+import styles from './common/DatePicker.test.module.scss';
+
+const cx = classNames.bind(styles);
+
+const DatePickerFieldDefault = () => (
```

Comment:
Looks like this test is not being used anywhere. Is it still needed? If anything, does https://github.com/cerner/terra-framework/blob/main/packages/terra-date-picker/src/terra-dev-site/test/date-picker/DatePickerField.test.jsx suffice?

Reply:
Removed the test here https://github.com/cerner/terra-framework/pull/1182/commits/28630bd2414aa22bb5f879c150fb4840bb3bc114